examples/hotrod/services/customer/database.go

```diff
@@ -73,15 +74,10 @@ func (d *database) Get(ctx context.Context, customerID string) (*Customer, error
 	d.logger.For(ctx).Info("Loading customer", zap.String("customer_id", customerID))
 
 	// simulate opentracing instrumentation of an SQL query
-	if span := opentracing.SpanFromContext(ctx); span != nil {
-		span := d.tracer.StartSpan("SQL SELECT", opentracing.ChildOf(span.Context()))
-		tags.SpanKindRPCClient.Set(span)
-		tags.PeerService.Set(span, "mysql")
-		// #nosec
-		span.SetTag("sql.query", "SELECT * FROM customer WHERE customer_id="+customerID)
-		defer span.Finish()
-		ctx = opentracing.ContextWithSpan(ctx, span)
-	}
+	ctx, span := d.tracer.Start(ctx, "SQL SELECT", trace.WithSpanKind(trace.SpanKindClient))
+	// #nosec
+	span.SetAttributes(semconv.RPCSystemKey.String("mysql"), attribute.Key("sql.query").String("SELECT * FROM customer WHERE customer_id=" + customerID))
```

Comment:
RPCSystem: A string identifying the remoting system (such as gRPC or java_rmi) - does not match

Reply:
Ah, sorry. my bad again. I wasn't sure if `PeerServiceKey` exists.